### ISSUE TITLE
OpenCover code coverage broken in 3.0 preview 4

### ISSUE THREAD

**Issue by martincostello** [dotnet/coreclr#24402]
I've just noticed in some 3.0 branches of some applications I'm maintaining as the previews ship, that code coverage metrics have stopped being reported with OpenCover in preview 4. The code coverage was being reported using preview 3.

Below are links to two commits of an application and its AppVeyor CI build showing code coverage being reported with preview 3, but not with preview 4. I've observed this with multiple applications, so I do not believe it is specific to this particular application.

| Release | Commit | Build |
|:-:|:-:|:-:|
| 3.0.100-preview3-010431 | https://github.com/martincostello/website/pull/254/commits/f1e71c056818cbc1d7470b3e8806745601dd6cd6 | [Logs](https://ci.appveyor.com/project/martincostello/website/builds/22879215#L218) |
| 3.0.100-preview4-011223 | https://github.com/martincostello/website/pull/254/commits/ac210c0baa6bd56a0826e275056928105107e5be | [Logs](https://ci.appveyor.com/project/martincostello/website/builds/23967773#L229) |

**Comment by RussKeldorph** [comment]
@martincostello Was this reported to OpenCover and determined to be a problem in CoreCLR?  Can you share more details?

/cc @danmosemsft

**Comment by martincostello** [comment]
@RussKeldorph No, I made the initial assumption that it was an issue in the CLR as the same version of OpenCover is otherwise working with 2.2 and 3.0 up to and including preview 3 and reporting coverage correctly.

If this is due to a considered profiling API change that creates an incompatibility, I'd be happy to log an issue over on OpenCover for getting it to adapt to any API changes required for when 3.0 ships in September.

**Comment by danmosemsft** [comment]
cc @ViktorHofer who owns code coverage for corefx.

**Comment by danmosemsft** [comment]
I misread- this is OpenCover. We moved to Coverlet in CoreFX.

**Comment by RussKeldorph** [comment]
@martincostello Unless @noahfalk knows something off hand, it's probably going to be quicker to have someone familiar with OpenCover investigate and determine what's different and whether it is an issue with OpenCover, CoreCLR, or something else.

**Comment by martincostello** [comment]
@RussKeldorph Sure, I'll post an issue over there now.

**Comment by martincostello** [comment]
https://github.com/OpenCover/opencover/issues/894

**Comment by noahfalk** [comment]
@davmason 

I'm not aware of anything deliberate that we would have done that would break OpenCover so unfortunately no easy answer offhand.
